backend/scripts/curl_bootstrap.sh

```
@@ -0,0 +1,36 @@
+echo 'version: '"'"'3.7'"'"'
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)